trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.test.ts

```diff
@@ -598,7 +600,7 @@ describe('controllers/utilisation-reports/utilisation-report-reconciliation-for-
         checkedCheckboxIds: {
           'feeRecordIds-1-reportedPaymentsCurrency-GBP-status-TO_DO': true,
         },
-        generateKeyingDataErrorKey: 'no-matching-fee-records',
```

Comment:
Could we also update `addPaymentErrorKey` within this `sessionData` object (line 599 at present) please to use the new constant seen as we're doing it elsewhere:

Suggested:
```
addPaymentErrorKey: ADD_PAYMENT_ERROR_KEY.NO_FEE_RECORDS_SELECTED
```

(It won't let me comment on the exact line as I think GH considers it as wider context and so is out of scope of the PR)

Reply:
Ah nice spot thanks!